odl/test/tomo/operators/ray_trafo_test.py

```diff
@@ -475,5 +476,138 @@ def test_shifted_volume(geometry_type):
     assert np.max(proj[3, 15:]) > 5
 
 
+def test_source_detector_shifts_2d():
+    """Check that source/detector shifts are handled correctly.
+
+    We forward project a Shepp-Logan phantom and check that reconstruction
+    with flying focal spot is equal to a sum of reconstructions with two
+    geometries which mimic ffs by using initial offsets
+    (the detector must be large enough, not to be influenced by shifts)
+    """
+
+    # If no implementation is available, skip
+    if not odl.tomo.ASTRA_AVAILABLE:
+        pytest.skip(msg='ASTRA not available, skipping 2d test')
```

Comment:
The fact that the test is skipped is documented automatically, I'd prefer information on why it was skipped. E.g. astra is needed to run it.